docs/sources/administration/api-keys/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'API keys in Grafana'
```

Comment:
@chri2547 @eleijonmarck shall we add a note about recommendation to use service accounts instead?

Reply:
yea, i think that would be a good idea. Do we have to mention why? or maybe just say that we recommend using service accounts and thats all for now.

We should say something, so il add some recommendation sentence here